src/ILCompiler/src/Program.cs

```diff
@@ -411,17 +411,28 @@ private int Run(string[] args)
                         }
                     }
 
+
+                    List<ModuleDesc> versionBubbleModules = new List<ModuleDesc>();
                     if (_isInputVersionBubble)
                     {
                         // In large version bubble mode add reference paths to the compilation group
-                        foreach (KeyValuePair<string, string> referenceFile in _referenceFilePaths)
+                        foreach (string referenceFile in _referenceFilePaths.Values)
                         {
-                            EcmaModule module = typeSystemContext.GetModuleFromPath(referenceFile.Value);
-                            inputModules.Add(module);
+                            try
+                            {
+                                EcmaModule module = typeSystemContext.GetModuleFromPath(referenceFile);
+                                versionBubbleModules.Add(module);
+
+                            }
+                            catch (Exception ex)
```

Comment:
What's the reasoning behind making a failure to resolve version bubble assemblies non-fatal?  Intuitively, I would think if a path is mis-specified we'd want to die early.

Reply:
As discussed offline, the problem is in the pre-existing script SimpleTest.targets that has no simple way of filtering out non-managed assemblies from the reference assembly set. In 2nd commit I have added a descriptive comment explaining this issue.

Reply:
Can we catch the exception specific to that? I think we would still want to not silently move on for access denied, etc.

Reply:
Good point, thanks, fixed in 4th commit.